Docker/Production/entando-full-stack/postgresql/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+export VERSION=${1:-5.0.1-SNAPSHOT}
```

Comment:
Should this be an argument?

Reply:
It kind of is... 5.0.1-SNAPSHOT is just the default value for this argument $1